docs/docsite/rst/command_guide/intro_adhoc.rst

```diff
@@ -198,6 +198,17 @@ Facts represent discovered variables about a system. You can use facts to implem
 
 You can also filter this output to display only certain facts, see the :ref:`ansible.builtin.setup <setup_module>` module documentation for details.
 
+Check mode
+----------
+
+In check mode, Ansible does not make any changes to remote systems. Ansible prints the commands only. It does not run the commands.
+
+.. code-block:: bash
+
+    $  ansible all -a "touch check.py" -C
```

Comment:
Hmm, I think using the command module (the default module) for a check mode test is a bad idea, since that will cause the task to simply be skipped as the command module does not support check mode.

How about this instead
```suggestion
    $  ansible all -m copy -a "content=foo dest=/root/bar.txt" -C
```
?